README.md

```diff
@@ -55,13 +55,14 @@ To download and install DreamFactory, follow the instructions [here](http://wiki
 ##Prerequisites
 
 - node v4.x.x higher and npm 2.14.7
+- Visual Studio (msbuild and vcbuild required by npm install on Windows)
```

Comment:
No, when you run npm install, VS is required to compile some of the libraries for windows.